[ci] corrupted keystore on java8 fips build

7.x periodic fips builds have been failing since 19 April with this, unsure if the master equivalent is also effected as it has not run since 7 April

```
org.elasticsearch.gradle.BuildExamplePluginsIT testCurrentExamplePlugin
org.elasticsearch.gradle.BuildExamplePluginsIT testCurrentExamplePlugin
org.elasticsearch.gradle.BuildExamplePluginsIT testCurrentExamplePlugin
org.elasticsearch.gradle.BuildExamplePluginsIT testCurrentExamplePlugin
org.elasticsearch.gradle.BuildExamplePluginsIT testCurrentExamplePlugin
org.elasticsearch.gradle.BuildExamplePluginsIT testCurrentExamplePlugin
org.elasticsearch.gradle.BuildExamplePluginsIT testCurrentExamplePlugin
```

```
  2> REPRODUCE WITH: ./gradlew :integTestRunner --tests "org.elasticsearch.example.painlesswhitelist.PainlessWhitelistClientYamlTestSuiteIT.test {yaml=painless_whitelist/30_static/custom static imported method}" -Dtests.seed=927E287605F73A70 -Dtests.security.manager=true -Dtests.locale=mt -Dtests.timezone=Egypt -Dcompiler.java=12 -Druntime.java=8FIPS
  2> java.lang.IllegalStateException: could not create the default ssl context

        Caused by:
        java.security.NoSuchAlgorithmException: Error constructing implementation (algorithm: Default, provider: SunJSSE, class: sun.security.ssl.SSLContextImpl$DefaultSSLContext)

            Caused by:
            java.security.KeyStoreException: problem accessing trust store

                Caused by:
                java.io.IOException: BCFKS KeyStore corrupted: MAC calculation failed.
```

Earliest https://scans.gradle.com/s/pvn346idu54go/tests/orjfcz7hu5lqq-o3tyn5hvqpucy
Most recent (slightly different stack trace) https://scans.gradle.com/s/kzcihhmmfxx54/tests/orjfcz7hu5lqq-axhth7m5yd5sq

The fix doesn't seem to have resolved the problem.

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+matrix-java-periodic/ES_BUILD_JAVA=openjdk12,ES_RUNTIME_JAVA=java8fips,nodes=immutable&&linux&&docker/191/

```
04:33:37 org.elasticsearch.gradle.JdkDownloadPluginIT > testWindowsExtractionOldVersion FAILED
04:33:37     java.security.KeyStoreException at JdkDownloadPluginIT.java:92
04:33:37         Caused by: java.io.IOException at JdkDownloadPluginIT.java:92
04:33:37             Caused by: java.security.UnrecoverableKeyException at JdkDownloadPluginIT.java:92
04:33:50 
04:33:50 org.elasticsearch.gradle.BuildExamplePluginsIT > testCurrentExamplePlugin {p0=/var/lib/jenkins/workspace/elastic+elasticsearch+7.x+matrix-java-periodic/ES_BUILD_JAVA/openjdk12/ES_RUNTIME_JAVA/java8fips/nodes/immutable&&linux&&docker/plugins/examples/custom-suggester} FAILED
04:33:50     org.gradle.testkit.runner.UnexpectedBuildFailure at BuildExamplePluginsIT.java:91
04:33:51 
```

Another failure:
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.1+matrix-java-periodic/ES_BUILD_JAVA=openjdk12,ES_RUNTIME_JAVA=java8fips,nodes=immutable&&linux&&docker/46/testReport/org.elasticsearch.gradle/BuildPluginIT/testLicenseAndNotice/

```
Caused by: org.gradle.api.resources.ResourceException: Could not get resource 'https://jcenter.bintray.com/junit/junit/4.12/junit-4.12.pom'.
	at org.gradle.internal.resource.ResourceExceptions.failure(ResourceExceptions.java:74)
	at org.gradle.internal.resource.ResourceExceptions.getFailed(ResourceExceptions.java:57)
	at org.gradle.internal.resource.transfer.DefaultCacheAwareExternalResourceAccessor.copyToCache(DefaultCacheAwareExternalResourceAccessor.java:201)
	at org.gradle.internal.resource.transfer.DefaultCacheAwareExternalResourceAccessor.access$300(DefaultCacheAwareExternalResourceAccessor.java:54)
	at org.gradle.internal.resource.transfer.DefaultCacheAwareExternalResourceAccessor$1.create(DefaultCacheAwareExternalResourceAccessor.java:89)
	at org.gradle.internal.resource.transfer.DefaultCacheAwareExternalResourceAccessor$1.create(DefaultCacheAwareExternalResourceAccessor.java:81)
	at org.gradle.cache.internal.ProducerGuard$AdaptiveProducerGuard.guardByKey(ProducerGuard.java:97)
	at org.gradle.internal.resource.transfer.DefaultCacheAwareExternalResourceAccessor.getResource(DefaultCacheAwareExternalResourceAccessor.java:81)
	at org.gradle.api.internal.artifacts.repositories.resolver.DefaultExternalResourceArtifactResolver.downloadByCoords(DefaultExternalResourceArtifactResolver.java:133)
	at org.gradle.api.internal.artifacts.repositories.resolver.DefaultExternalResourceArtifactResolver.downloadStaticResource(DefaultExternalResourceArtifactResolver.java:97)
	at org.gradle.api.internal.artifacts.repositories.resolver.DefaultExternalResourceArtifactResolver.resolveArtifact(DefaultExternalResourceArtifactResolver.java:64)
	at org.gradle.api.internal.artifacts.repositories.metadata.AbstractRepositoryMetadataSource.parseMetaDataFromArtifact(AbstractRepositoryMetadataSource.java:69)
	at org.gradle.api.internal.artifacts.repositories.metadata.AbstractRepositoryMetadataSource.create(AbstractRepositoryMetadataSource.java:59)
	at org.gradle.api.internal.artifacts.repositories.metadata.DefaultMavenPomMetadataSource.create(DefaultMavenPomMetadataSource.java:38)
	at org.gradle.api.internal.artifacts.repositories.resolver.ExternalResourceResolver.resolveStaticDependency(ExternalResourceResolver.java:243)
	at org.gradle.api.internal.artifacts.repositories.resolver.MavenResolver.doResolveComponentMetaData(MavenResolver.java:126)
	at org.gradle.api.internal.artifacts.repositories.resolver.ExternalResourceResolver$RemoteRepositoryAccess.resolveComponentMetaData(ExternalResourceResolver.java:444)
	at org.gradle.api.internal.artifacts.ivyservice.ivyresolve.CachingModuleComponentRepository$ResolveAndCacheRepositoryAccess.resolveComponentMetaData(CachingModuleComponentRepository.java:373)
	at org.gradle.api.internal.artifacts.ivyservice.ivyresolve.ErrorHandlingModuleComponentRepository$ErrorHandlingModuleComponentRepositoryAccess.lambda$resolveComponentMetaData$3(ErrorHandlingModuleComponentRepository.java:159)
	at org.gradle.api.internal.artifacts.ivyservice.ivyresolve.ErrorHandlingModuleComponentRepository$ErrorHandlingModuleComponentRepositoryAccess.lambda$tryResolveAndMaybeBlacklist$15(ErrorHandlingModuleComponentRepository.java:228)
	at org.gradle.api.internal.artifacts.ivyservice.ivyresolve.ErrorHandlingModuleComponentRepository$ErrorHandlingModuleComponentRepositoryAccess.tryResolveAndMaybeBlacklist(ErrorHandlingModuleComponentRepository.java:242)
	... 102 more
Caused by: com.google.common.util.concurrent.UncheckedExecutionException: org.apache.http.ssl.SSLInitializationException: Keystore was tampered with, or password was incorrect
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2048)
	at com.google.common.cache.LocalCache.get(LocalCache.java:3850)
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3874)
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4799)
	at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4805)
	at org.gradle.internal.resource.transport.http.DefaultSslContextFactory.createSslContext(DefaultSslContextFactory.java:58)
	at org.gradle.internal.resource.transport.http.HttpClientConfigurer.configureSslSocketConnectionFactory(HttpClientConfigurer.java:122)
	at org.gradle.internal.resource.transport.http.HttpClientConfigurer.configure(HttpClientConfigurer.java:107)
	at org.gradle.internal.resource.transport.http.HttpClientHelper.getClient(HttpClientHelper.java:169)
	at org.gradle.internal.resource.transport.http.HttpClientHelper.performHttpRequest(HttpClientHelper.java:132)
	at org.gradle.internal.resource.transport.http.HttpClientHelper.performHttpRequest(HttpClientHelper.java:109)
	at org.gradle.internal.resource.transport.http.HttpClientHelper.executeGetOrHead(HttpClientHelper.java:98)
	at org.gradle.internal.resource.transport.http.HttpClientHelper.performRequest(HttpClientHelper.java:89)
	at org.gradle.internal.resource.transport.http.HttpClientHelper.performRawGet(HttpClientHelper.java:76)
	at org.gradle.internal.resource.transport.http.HttpClientHelper.performGet(HttpClientHelper.java:80)
	at org.gradle.internal.resource.transport.http.HttpResourceAccessor.openResource(HttpResourceAccessor.java:42)
	at org.gradle.internal.resource.transport.http.HttpResourceAccessor.openResource(HttpResourceAccessor.java:28)
	at org.gradle.internal.resource.transfer.DefaultExternalResourceConnector.openResource(DefaultExternalResourceConnector.java:56)
	at org.gradle.internal.resource.transfer.ProgressLoggingExternalResourceAccessor.openResource(ProgressLoggingExternalResourceAccessor.java:37)
	at org.gradle.internal.resource.transfer.AccessorBackedExternalResource.withContentIfPresent(AccessorBackedExternalResource.java:130)
	at org.gradle.internal.resource.BuildOperationFiringExternalResourceDecorator$11.call(BuildOperationFiringExternalResourceDecorator.java:237)
	at org.gradle.internal.resource.BuildOperationFiringExternalResourceDecorator$11.call(BuildOperationFiringExternalResourceDecorator.java:229)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:416)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:406)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:102)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.call(DelegatingBuildOperationExecutor.java:36)
	at org.gradle.internal.resource.BuildOperationFiringExternalResourceDecorator.withContentIfPresent(BuildOperationFiringExternalResourceDecorator.java:229)
	at org.gradle.internal.resource.transfer.DefaultCacheAwareExternalResourceAccessor.copyToCache(DefaultCacheAwareExternalResourceAccessor.java:199)
	... 120 more
Caused by: org.apache.http.ssl.SSLInitializationException: Keystore was tampered with, or password was incorrect
	at org.gradle.internal.resource.transport.http.DefaultSslContextFactory$SslContextCacheLoader.load(DefaultSslContextFactory.java:183)
	at org.gradle.internal.resource.transport.http.DefaultSslContextFactory$SslContextCacheLoader.load(DefaultSslContextFactory.java:75)
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3444)
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2193)
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2152)
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2042)
	... 150 more
Caused by: java.io.IOException: Keystore was tampered with, or password was incorrect
	at org.gradle.internal.resource.transport.http.DefaultSslContextFactory$SslContextCacheLoader.load(DefaultSslContextFactory.java:128)
	... 155 more
Caused by: java.security.UnrecoverableKeyException: Password verification failed
```

Another failure in https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.1+matrix-java-periodic/ES_BUILD_JAVA=openjdk12,ES_RUNTIME_JAVA=java8fips,nodes=immutable&&linux&&docker/54/console:

```
Caused by: org.gradle.api.resources.ResourceException: Could not get resource 'https://jcenter.bintray.com/org/locationtech/spatial4j/spatial4j/0.7/spatial4j-0.7.pom'.
	at org.gradle.internal.resource.ResourceExceptions.failure(ResourceExceptions.java:74)
	at org.gradle.internal.resource.ResourceExceptions.getFailed(ResourceExceptions.java:57)
[...]
	at org.gradle.api.internal.artifacts.ivyservice.ivyresolve.ErrorHandlingModuleComponentRepository$ErrorHandlingModuleComponentRepositoryAccess.tryResolveAndMaybeBlacklist(ErrorHandlingModuleComponentRepository.java:242)
	... 31 more
Caused by: com.google.common.util.concurrent.UncheckedExecutionException: org.apache.http.ssl.SSLInitializationException: Keystore was tampered with, or password was incorrect
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2048)
[...]
	at org.gradle.internal.resource.BuildOperationFiringExternalResourceDecorator.withContentIfPresent(BuildOperationFiringExternalResourceDecorator.java:229)
	at org.gradle.internal.resource.transfer.DefaultCacheAwareExternalResourceAccessor.copyToCache(DefaultCacheAwareExternalResourceAccessor.java:199)
	... 49 more
Caused by: org.apache.http.ssl.SSLInitializationException: Keystore was tampered with, or password was incorrect
	at org.gradle.internal.resource.transport.http.DefaultSslContextFactory$SslContextCacheLoader.load(DefaultSslContextFactory.java:183)
	at org.gradle.internal.resource.transport.http.DefaultSslContextFactory$SslContextCacheLoader.load(DefaultSslContextFactory.java:75)
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3444)
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2193)
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2152)
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2042)
	... 79 more
Caused by: java.io.IOException: Keystore was tampered with, or password was incorrect
	at org.gradle.internal.resource.transport.http.DefaultSslContextFactory$SslContextCacheLoader.load(DefaultSslContextFactory.java:128)
	... 84 more
Caused by: java.security.UnrecoverableKeyException: Password verification failed
	... 85 more
```

<details>
    <summary>Failed tests</summary>
    <ul>
<li>org.elasticsearch.gradle.BuildExamplePluginsIT.testCurrentExamplePlugin {p0=/var/lib/jenkins/workspace/elastic+elasticsearch+7.1+matrix-java-periodic/ES_BUILD_JAVA/openjdk12/ES_RUNTIME_JAVA/java8fips/nodes/immutable&&linux&&docker/plugins/examples/script-expert-scoring}</li>
<li>org.elasticsearch.gradle.BuildExamplePluginsIT.testCurrentExamplePlugin {p0=/var/lib/jenkins/workspace/elastic+elasticsearch+7.1+matrix-java-periodic/ES_BUILD_JAVA/openjdk12/ES_RUNTIME_JAVA/java8fips/nodes/immutable&&linux&&docker/plugins/examples/custom-suggester}</li>
<li>org.elasticsearch.gradle.BuildExamplePluginsIT.testCurrentExamplePlugin {p0=/var/lib/jenkins/workspace/elastic+elasticsearch+7.1+matrix-java-periodic/ES_BUILD_JAVA/openjdk12/ES_RUNTIME_JAVA/java8fips/nodes/immutable&&linux&&docker/plugins/examples/rest-handler}</li>
<li>org.elasticsearch.gradle.BuildExamplePluginsIT.testCurrentExamplePlugin {p0=/var/lib/jenkins/workspace/elastic+elasticsearch+7.1+matrix-java-periodic/ES_BUILD_JAVA/openjdk12/ES_RUNTIME_JAVA/java8fips/nodes/immutable&&linux&&docker/plugins/examples/rescore}</li>
<li>org.elasticsearch.gradle.BuildExamplePluginsIT.testCurrentExamplePlugin {p0=/var/lib/jenkins/workspace/elastic+elasticsearch+7.1+matrix-java-periodic/ES_BUILD_JAVA/openjdk12/ES_RUNTIME_JAVA/java8fips/nodes/immutable&&linux&&docker/plugins/examples/security-authorization-engine}</li>
<li>org.elasticsearch.gradle.BuildExamplePluginsIT.testCurrentExamplePlugin {p0=/var/lib/jenkins/workspace/elastic+elasticsearch+7.1+matrix-java-periodic/ES_BUILD_JAVA/openjdk12/ES_RUNTIME_JAVA/java8fips/nodes/immutable&&linux&&docker/plugins/examples/painless-whitelist}</li>
<li>org.elasticsearch.gradle.BuildExamplePluginsIT.testCurrentExamplePlugin {p0=/var/lib/jenkins/workspace/elastic+elasticsearch+7.1+matrix-java-periodic/ES_BUILD_JAVA/openjdk12/ES_RUNTIME_JAVA/java8fips/nodes/immutable&&linux&&docker/plugins/examples/custom-settings}</li>
<li>org.elasticsearch.gradle.BuildPluginIT.testCheckTask</li>
<li>org.elasticsearch.gradle.BuildPluginIT.testLicenseAndNotice</li>
<li>org.elasticsearch.gradle.precommit.JarHellTaskIT.testJarHellDetected</li>
<li>org.elasticsearch.gradle.precommit.TestingConventionsTasksIT.testAllTestTasksIncluded</li>
<li>org.elasticsearch.gradle.precommit.TestingConventionsTasksIT.testValidSetupWithoutBaseClass</li>
<li>org.elasticsearch.gradle.precommit.TestingConventionsTasksIT.testNamingConvention</li>
<li>org.elasticsearch.gradle.precommit.TestingConventionsTasksIT.testTestsInMain</li>
<li>org.elasticsearch.gradle.precommit.TestingConventionsTasksIT.testTaskNotImplementBaseClass</li>
<li>org.elasticsearch.gradle.precommit.TestingConventionsTasksIT.testValidSetupWithBaseClass</li>
<li>org.elasticsearch.gradle.precommit.TestingConventionsTasksIT.testInnerClasses</li>
<li>org.elasticsearch.gradle.precommit.TestingConventionsTasksIT.testNoEmptyTasks</li>
<li>org.elasticsearch.gradle.precommit.ThirdPartyAuditTaskIT.testJarHellWithJDK</li>
<li>org.elasticsearch.gradle.precommit.ThirdPartyAuditTaskIT.testElasticsearchIgnored</li>
<li>org.elasticsearch.gradle.precommit.ThirdPartyAuditTaskIT.testViolationFoundAndCompileOnlyIgnored</li>
<li>org.elasticsearch.gradle.precommit.ThirdPartyAuditTaskIT.testWithEmptyRules</li>
<li>org.elasticsearch.gradle.precommit.ThirdPartyAuditTaskIT.testClassNotFoundAndCompileOnlyIgnored</li>
<li>org.elasticsearch.gradle.precommit.ThirdPartyAuditTaskIT.testElasticsearchIgnoredWithViolations</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testMultiProject</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testConfigurationLocked</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testUseClusterByUpToDateTask</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testUseClusterByFailingOne</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testUseClusterBySkippedTask</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testUseClusterByOne</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testUseClusterBySkippedAndWorkingTask</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testUseClusterByFailingDependency</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testPluginInstalled</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testUseClusterByTwo</li>
<li>org.elasticsearch.gradle.testclusters.TestClustersPluginIT.testIncremental</li>
    </ul>
</details>

A slightly different failure with the same cause:
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+matrix-java-periodic/ES_BUILD_JAVA=openjdk12,ES_RUNTIME_JAVA=java8fips,nodes=immutable&&linux&&docker/215/console
```
04:39:55   Caused by: java.io.IOException: BCFKS KeyStore corrupted: MAC calculation failed.
04:39:55   	at org.bouncycastle.jcajce.provider.ProvBCFKS$BCFIPSKeyStoreSpi.verifyMac(Unknown Source) ~[bc-fips-1.0.1.jar:?]
04:39:55   	at org.bouncycastle.jcajce.provider.ProvBCFKS$BCFIPSKeyStoreSpi.engineLoad(Unknown Source) ~[bc-fips-1.0.1.jar:?]
04:39:55   	at java.security.KeyStore.load(KeyStore.java:1445) ~[?:1.8.0_212]
04:39:55   	at sun.security.ssl.TrustStoreManager$TrustAnchorManager.loadKeyStore(TrustStoreManager.java:368) ~[?:?]
04:39:55   	at sun.security.ssl.TrustStoreManager$TrustAnchorManager.getTrustedCerts(TrustStoreManager.java:316) ~[?:?]
04:39:55   	at sun.security.ssl.TrustStoreManager.getTrustedCerts(TrustStoreManager.java:59) ~[?:?]
04:39:55   	at sun.security.ssl.TrustManagerFactoryImpl.engineInit(TrustManagerFactoryImpl.java:51) ~[?:?]
04:39:55   	at javax.net.ssl.TrustManagerFactory.init(TrustManagerFactory.java:250) ~[?:1.8.0_212]
04:39:55   	at org.elasticsearch.xpack.core.ssl.CertParsingUtils.trustManager(CertParsingUtils.java:279) ~[?:?]
04:39:55   	at org.elasticsearch.xpack.core.ssl.DefaultJDKTrustConfig.createTrustManager(DefaultJDKTrustConfig.java:48) ~[?:?]

```

```
04:39:55 > Task :docs:integTestRunner FAILED
04:39:55 
04:39:55 FAILURE: Build failed with an exception.
04:39:55 
04:39:55 * What went wrong:
04:39:55 Execution failed for task ':docs:integTestRunner'.
04:39:55 > process was found dead while waiting for ports files, node{:docs:integTest-0}
04:39:55 
```

--- a/buildSrc/build.gradle
+++ b/buildSrc/build.gradle
@@ -182,7 +182,9 @@ if (project != rootProject) {
     if (isLuceneSnapshot) {
       systemProperty 'test.lucene-snapshot-revision', isLuceneSnapshot[0][1]
     }
-    maxParallelForks System.getProperty('tI', project.rootProject.ext.defaultParallel.toString()) as Integer
+    maxParallelForks System.getProperty('tests.jvms', project.rootProject.ext.defaultParallel.toString()) as Integer
+    // These tests run Gradle which doesn't have FIPS support
+    onlyIf { project.inFipsJvm == false }
   }
   check.dependsOn(integTest)
 

